.kokoro/tests/run_single_test.sh

```diff
@@ -41,6 +41,21 @@ fi
 nox -s "$RUN_TESTS_SESSION"
 EXIT=$?
 
+# Inject region tag data into the test log
+if [[ "${INJECT_REGION_TAGS:-}" == "true" ]]; then
+    echo "=== Injecting region tags into XUnit output ==="
+    XUNIT_PATH="$PWD/sponge_log.xml"
+    XUNIT_TMP_PATH="tmp/drift_tmp.xml"
```

Comment:
I don't think there is `tmp` directory.
How about:
```
XUNIT_TMP_PATH=$(mktemp ${HOME}/xunit-tmp.xxxxx)
```
 or something? The trampoline script will clean up ${HOME} upon exit, so you don't have to clean up on your own.

Reply:
AFAIK `/tmp` is a pretty standard directory on Linux.

(I fixed the missing `/`)

Reply:
It will work, but here is a reason I prefer ${HOME} over `/tmp`.

Currently Trampoline mount host's `/tmp` under `/tmp` for an unrelated reason, so if you create files under `/tmp`, the files will be remain in the host's `/tmp` directory.

Probably we should change this, but this is how it works.

Whereas, files in ${HOME} will be automatically clean up by Trampoline.

Reply:
Gotcha. I'll change this to `$HOME`, and keep the _parsing script itself_ in `tmp` (since that can be reused between invocations).